src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +70,65 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
+            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
+            {
+                if (regNsbRoot == null)
+                {
+                    //Check for the presence of HKCU:SOFTWARE\ParticularSoftware
+                    using (var regParticularRoot = Registry.CurrentUser.OpenSubKey(@"Software\ParticularSoftware"))
+                    {
+                        if (regParticularRoot == null)
+                        {
+                            return true;
+                        }
+                    }
+                }
+            }
+            return false;
+        }
+
+
+        private void TrackFirstTimeUsageEvent()
+        {
+            // Get the current version of NServiceBus that's being used
+            var assembly = Assembly.GetExecutingAssembly();
+            var version = assembly.GetName().Version.ToString();
+
+            if (assembly.Location != null)
+            {
+                version = FileVersionInfo.GetVersionInfo(assembly.Location).FileVersion;
```

Comment:
To avoid getting a version that has the extra 4th 0 on the end, I'd recommend doing something like:

```csharp
var fileVersionInfo = FileVersionInfo.GetVersionInfo(assembly.Location);
version = $"{fileVersionInfo.FileMajorPart}.{fileVersionInfo.FileMinorPart}.{fileVersionInfo.FileBuildPart}"
```
That will have just the 3 parts we care about.